packages/lab/src/calendar/internal/CalendarDay.tsx

```diff
@@ -58,9 +60,10 @@ export const CalendarDay = forwardRef<HTMLButtonElement, CalendarDayProps>(
         {
           [withBaseName("hidden")]: hidden,
           [withBaseName("outOfRange")]: outOfRange,
-          [withBaseName("unselectableHighEmphasis")]: unselectable === "high",
           [withBaseName("today")]: today,
           [withBaseName("unselectableLowEmphasis")]: unselectable === "low",
+          [withBaseName("unselectableMediumEmphasis")]:
```

Comment:
to match our conventions could we make this uitkEmphasisMedium/Low as usual, and selector .uitkCalendarDay.uitkEmphasisMedium/Low or .uitkCalendarDay-unselectable.uitkEmphasisMedium/Low ?